photon-client/src/components/dashboard/CameraAndPipelineSelectCard.vue

```diff
@@ -0,0 +1,425 @@
+          :items="[
+            {name: 'Reflective', value: WebsocketPipelineType.Reflective},
+            {name: 'Colored Shape', value: WebsocketPipelineType.ColoredShape},
+            {name: 'AprilTag', value: WebsocketPipelineType.AprilTag}]"
+          @input="showPipelineTypeChangeDialog = true"
+        />
+      </v-col>
+    </v-row>
+    <v-dialog
+      v-model="showPipelineCreationDialog"
+      dark
+      persistent
+      width="500"
+    >
+      <v-card
+        dark
+        color="primary"
+      >
+        <v-card-title
+          class="headline"
+          style="font-family: 'Prompt', sans-serif !important;"
+          primary-title
+        >
+          Create New Pipeline
+        </v-card-title>
+        <v-card-text>
+          <cv-input
+            v-model="newPipelineName"
+            placeholder="Pipeline Name"
+            :label-cols="3"
+            :input-cols="12-3"
+            label="Pipeline Name"
+            :rules="[v => checkPipelineName(v)]"
+          />
+          <cv-select
+            v-model="newPipelineType"
+            :select-cols="12-3"
+            label="Tracking Type"
+            tooltip="Pipeline type, which changes the type of processing that will happen on input frames"
+            :items="[
+              {name: 'Reflective', value: WebsocketPipelineType.Reflective},
+              {name: 'Colored Shape', value: WebsocketPipelineType.ColoredShape},
+              {name: 'AprilTag', value: WebsocketPipelineType.AprilTag}]"
+          />
+        </v-card-text>
+        <v-divider />
+        <v-card-actions>
+          <v-spacer />
+          <v-btn
+            color="#ffd843"
+            :disabled="checkPipelineName(newPipelineName) !== true"
+            @click="createNewPipeline"
+          >
+            Save
+          </v-btn>
+          <v-btn
+            color="error"
+            @click="cancelPipelineCreation"
+          >
+            Cancel
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+    <v-dialog
+      v-model="showPipelineDeletionConfirmationDialog"
+      dark
+      width="500"
+    >
+      <v-card
+        dark
+        color="primary"
+      >
+        <v-card-title
+          class="headline"
+          style="font-family: 'Prompt', sans-serif !important;"
+          primary-title
+        >
+          Pipeline Deletion Confirmation
+        </v-card-title>
+        <v-card-text>
+          Are you sure you want to delete this pipeline? This cannot be undone.
+        </v-card-text>
+        <v-divider />
+        <v-card-actions>
+          <v-spacer />
+          <v-btn
+            color="error"
+            @click="confirmDeleteCurrentPipeline"
+          >
+            Yes, Im Sure
+          </v-btn>
+          <v-btn
+            color="#ffd843"
+            @click="showPipelineDeletionConfirmationDialog = false"
+          >
+            No, Take me Back
```

Comment:
```suggestion
            No, take me back
```

I realize it ends up all caps regardless